src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java

```diff
@@ -30,20 +45,47 @@ public boolean hasTarget() {
         return limelight.getEntry("tv").getDouble(0) == 1;
     }
 
-    // x-axis
+    /**
+     * Returns the yaw from the limelight to the hub.
+     *
+     * @return The yaw (horizontal rotation) in degrees.
+     */
     public double getYaw() {
         return limelight.getEntry("tx").getDouble(0);
     }
 
-    // returns in inches
+    /**
+     * Returns the distance from the limelight to the hub.
+     *
+     * @return THe distance in inches.
+     */
     public double getDistance() {
-        double distance = HEIGHT_TO_RIM / Math.tan(LIMELIGHT_ANGLE_OFFSET + getPitch());
-        return distance;
+        return HEIGHT_TO_RIM / Math.tan(Math.toRadians(LIMELIGHT_ANGLE_OFFSET + getPitch()));
```

Comment:
Note that this will find the distance to the vision target and not to the center of the hub. The latter might be more useful as it's a fixed point whose location doesn't change based on where the robot is.